python/cudf/cudf/tests/test_column.py

```diff
@@ -255,36 +255,29 @@ def test_column_view_nulls_widths_even():
 
     assert_eq(expect, got)
 
-
-def test_column_view_numeric_slice():
+@pytest.mark.parametrize('slc', [slice(1, None), slice(None, 4), slice(2, 4)])
+def test_column_view_numeric_slice(slc):
 
     data = np.array([1, 2, 3, 4, 5], dtype="int32")
     sr = cudf.Series(data)
 
-    expect = cudf.Series(data[1:].view("int64"))
-    got = cudf.Series(sr._column[1:].view("int64"))
+    expect = cudf.Series(data[slc].view("int64"))
+    got = cudf.Series(sr._column[slc].view("int64"))
 
     assert_eq(expect, got)
 
-    expect = cudf.Series(data[1:3])
-    got = cudf.Series(sr._column[1:3].view("int64"))
-
-def test_column_view_string_slice():
+@pytest.mark.parametrize('slc', [slice(3, None), slice(None, 4), slice(2 , 5)])
```

Comment:
Can we add `slice(1,3)` here to test where we slice off from both the beginning and end? `slice(2, 5)` doesn't slice off the end currently.